<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame {
background-color:#Dbdcd8;
border-radius:5px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="txtTitulo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#txtTitulo{
background-color:transparent;
border:none;
border-bottom:1px solid gray;
padding:5px;
border-radius:3px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Titulo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNotas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>301</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#txtNotas{
background-color:transparent;
border:none;
border-bottom:1px solid gray;
padding:5px;
border-radius:3px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Nota</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="txtDt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#txtDt{
background-color:transparent;
border:none;
border-bottom:1px solid gray;
padding:5px;

}</string>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2024</year>
       <month>4</month>
       <day>18</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::TimeZone</enum>
     </property>
     <property name="date">
      <date>
       <year>2024</year>
       <month>4</month>
       <day>18</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="lblEscolha">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblEscolha{
color:#484848;
font-family:sans-serif;
border-radius:5px;
}</string>
     </property>
     <property name="text">
      <string>Escolha uma data</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#comboBox{
padding:5px;
}</string>
     </property>
     <item>
      <property name="text">
       <string>Programados</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Concluidos</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lblStatus">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblStatus{
color:#484848;
font-family:sans-serif;
border-radius:5px;
}</string>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnNovo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sans-serif</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnNovo{
color:#484848;
font-family:sans-serif;
border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>+ Novo Lembrete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sans-serif</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnCancelar{
color:#484848;
font-family:sans-serif;
border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>- Cancelar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
